src/user_lists/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+from utils.serializers import DefaultAuthenticatedSerializer
+
+from .models import List
+
+
+class ListSerializer(DefaultAuthenticatedSerializer):
+    class Meta:
+        model = List
+        fields = [
+            "id",
+            "name",
+            "is_public",
+            "created_date",
+            "updated_date",
+            "created_by",
+            "updated_by",
```

Comment:
I don't think `updated_by` is a thing. Let's remove if it is is since we don't need to keep track of it